style.css

```diff
@@ -547,3 +547,494 @@ hr {
   border-radius: 10px;
   background-color: black;
 }
+
+.title hr {
+  width: 100%;
+  height: 1px;
+}
+
+/* desktop-version */
+
+@media only screen and (min-width: 760px) {
```

Comment:
- [ ] Hi the general design is really good. However, I'd like you to notice that this separator above the social media icons is too dark compared to the Figma template. Please address this issue and try to match the Figma as close as possible
## Your Design
![image](https://user-images.githubusercontent.com/18730106/145456241-ffba1e87-7510-4f84-b420-59501d655591.png)

## Figma Design
![image](https://user-images.githubusercontent.com/18730106/145456194-41546cd5-a4b9-4621-bb0c-ca86cc434463.png)